test/verify/check-system-services

```diff
@@ -530,28 +530,33 @@ ExecStart=/bin/sh -c "while true; do sleep 5; done"
 
         # service a
         b.wait_js_cond('window.location.hash === "#/test-a.service"')
+        b.click("#service-details-show-relationships  button")
```

Comment:
double space (Sanne would be *so* proud! :grin:)

Reply:
Fixed